docs/migration-guide/configs-conversion-guide.md

```diff
@@ -0,0 +1,73 @@
+
+Below is a summary of how to populate the fields of an FPL worker configuration file based on the state of your existing CPL project:
+
+| FPL format | CPL format |
+| --- | --- |
+| `workerType` | Part of the name of your worker configuration file, i.e. `spatialos.<my-worker-type>.worker.json` |
+| `layer` | In your worker configuration file, the `bridge/worker_attribute_set/attributes` field |
+| `entityInterest` | `bridge/entity_interest` field in your worker configuration file |
+| `streamingQuery` | `bridge/streaming_query` field in your worker configuration file |
+| `componentDelivery` | `bridge/component_delivery` field in your worker configuration file |
+| `componentSettings` | `bridge/component_settings` field in your worker configuration file |
+| `permissions` | Similar to the `workers/<my-worker-type>/permissions` field in your [launch configuration file](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config#launch-configuration-file), with the difference that the permissions property here is a single [permission object](https://docs.improbable.io/reference/latest/shared/worker-configuration/permissions#worker-permissions) rather than an array |
+| `uploadConfiguration/<platform>/localBuildDirectory` (client-worker only) | No equivalent - refer to [the documentation](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/reference/client-worker-configuration.md) for information on how to populate this field |
+| `launchConfiguration/<platform>/localBuildDirectory` (server-worker only) | No equivalent - refer to [the documentation](https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/reference/client-worker-configuration.md) for information on how to populate this field |
+| `launchConfiguration/<localDeployment, cloudDeployment>/<platform>/<command, arguments>` (server-worker only) | `managed/<platform>/<command, arguments>` field in you worker configuration file |
+
+**Notes:**
+- The `build` field in the CPL worker configuration is obsolete and does not have an equivalent in the FPL. In the FPL, you fully manage the build process of a worker. For more information, see [Building a worker executable in the FPL](../build-process/worker-build-process.md).
+- The `external` field in the CPL worker configuration is obsolete and does not have an equivalent in the FPL. In the FPL, you need to start a worker manually instead of using `spatial local worker launch`.
```

Comment:
```suggestion
- The `external` field in the CPL worker configuration file is obsolete and does not have an equivalent in the FPL. In the FPL, you need to start a worker manually instead of using `spatial local worker launch`.
```

Reply:
Done.